website/docs/components/app-header/partials/code/how-to-use.md

```diff
@@ -0,0 +1,177 @@
+!!! Warning
+
+The `AppHeader` is being rolled out in a phased approach. In the phase one release, we will assist consumers in adoption spikes to optimize it to their needs. This unique approach is due to how the `AppHeader` tightly integrates into the release plan for the “Enterprise navigation” which includes the `SideNav` and `AppFrame` components.
+
+At this time, we do not recommend adoption on your own without the assistance of HDS team members.
+
+!!!
+
+## How to use this component
+
+The `AppHeader` provides persistent global navigation controls and utility links such as to help and user menus. It is meant to be paired with the `SideNav` which provides local or page-level navigation.
+
+!!! Insight
```

Comment:
Why this is inside a banner? We're in a context of code implementation, so I think the content of this banner and the previous paragraph should be merged.

Reply:
removing